src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltTransectInputs.js

```diff
@@ -72,6 +74,8 @@ const FishBeltTransectInputs = ({
   // account for empty starter row
   const hasFishBeltObservations =
     !!observationsState.length > 0 && observationsState[0]?.fish_attribute
+  const [isClearSizeValueModalOpen, setIsClearSizeValueModalOpen] = useState(false)
+  const [sizeBinEvent, setSizeBinEvent] = useState({})
```

Comment:
Its weird that we are storing an whole event, but I get you on refactor tradeoffs. (Im actually passing around whole events on my other project for timeline tradeoffs, but I dont like it)

Reply:
yeah it feels like a tradeoff but since we're using it in `resetSizeValues` function, decided it was worth keeping for now